string station
-------
bool success
string connection_status #To show connection status